Description: do not add timestamps to documentation
Author: Helmut Grohne
Forwarded: https://github.com/doxygen/doxygen/issues/5751
Last-Update: 2019-08-13

In order to improve reproducibility of the doxygen-doc package, elide
timestamps from the generated HTML documentation.

Index: doxygen/Doxyfile
===================================================================
--- doxygen.orig/Doxyfile
+++ doxygen/Doxyfile
@@ -157,7 +157,7 @@ HTML_EXTRA_FILES       =
 HTML_COLORSTYLE_HUE    = 220
 HTML_COLORSTYLE_SAT    = 100
 HTML_COLORSTYLE_GAMMA  = 80
-HTML_TIMESTAMP         = YES
+HTML_TIMESTAMP         = NO
 HTML_DYNAMIC_MENUS     = YES
 HTML_DYNAMIC_SECTIONS  = YES
 HTML_INDEX_NUM_ENTRIES = 100
